tests/qml/main.cpp

```diff
@@ -0,0 +1,11 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+#include "helper.h"
+#include <QtQuickTest>
+#include "../../src/settingsholder.h"
+
+SettingsHolder settingsHolder;
```

Comment:
Can this be moved into `TestHelper` as a member var?

Reply:
NO! Everything breaks. 

It *may* be that now I've finally got everything it works but I don't think so.

Reply:
it works now :D